app/(route)/playground/[uid]/_nav.tsx

```
@@ -0,0 +1,260 @@
+            key={index}
+            className={cx('flex items-center justify-center rounded-[5px] border-[7px] border-transparent text-center text-[35px] hover:border-white', {
+              'bg-white': selectedPieceNumber === pieceNumber,
+            })}
+            onClick={() => handleClickPieceNumber(pieceNumber)}
+            onKeyDown={(e) => {
+              if (e.key === 'Enter') {
+                handleClickPieceNumber(pieceNumber);
+              }
+            }}
+            tabIndex={0}
+            role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)